src/main/java/mate/controller/GetManufacturersController.java

```diff
@@ -0,0 +1,23 @@
+package mate.controller;
+
+import java.io.IOException;
+import javax.servlet.ServletException;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import mate.lib.Injector;
+import mate.service.ManufacturerService;
+
+public class GetManufacturersController extends HttpServlet {
+    private static final String MANUFACTURERS_ATTRIBUTE = "manufacturers";
```

Comment:
what the purpose of this constant? constants should be reusable, no need to make them if there is no reusage.